src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -126,9 +144,11 @@ public VirtualProjectBuildingCommand(
     public string EntryPointFileFullPath { get; }
     public MSBuildArgs MSBuildArgs { get; }
     public string? CustomArtifactsPath { get; init; }
+    public string ArtifactsPath => field ??= CustomArtifactsPath ?? GetArtifactsPath(EntryPointFileFullPath);
```

Comment:
does `CustomArtifactsPath` mean something like: `dotnet run app.cs -o /path/to/artifacts`?